docs/buildrun.md

```diff
@@ -213,6 +215,46 @@ To make it easier for users to understand why did a BuildRun failed, users can i
 
 In addition, the `Status.Conditions` will host under the `Message` field a compacted message containing the `kubectl` command to trigger, in order to retrieve the logs.
 
+### Step Results in BuildRun Status
+
+After the successful completion of a `BuildRun`, the `.status` field contains the results (`.status.taskResults`) emitted from the `TaskRun` steps. These results contain valuable metadata for users, like the _image digest_ or the _commit sha_ of the source code used for building.
```

Comment:
nit:

emitted from the `TaskRun` steps. -> emitted from the `TaskRun` steps generate by the `BuildRun` controller as part of processing the `BuildRun`.